project.py

```diff
@@ -106,7 +106,8 @@ def area_code_lookup(phone_nums:str) -> dict:
     output_dict = {}
     for num in phone_list:
         area_code = num[0:3]
-        if ~bool(re.match(num, '/\d{3}-\d{3}-\d{4})/')) & (int(area_code) > 200):
+        #fixed regex and match statement

```

Comment:
Comments in the code are more for explaining how parts of it work, and the fix was explained in your commit comment, so this comment doesn't seem necessary.